src/test-interp.cc

```diff
@@ -718,6 +718,13 @@ TEST_F(InterpGCTest, Collect_DeepRecursion) {
   EXPECT_EQ(1u, store_.object_count());
 }
 
+TEST_F(InterpGCTest, Store_Assignemnt) {
+  wabt::interp::Store store;
+
+  store = wabt::interp::Store();
```

Comment:
I think you can do this all one line no?

Reply:
Should this just be `store_ = wabt::interp::Store();`  (note the trailing underscore).   Since the assertion below relates to the member `store_`?